backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -295,3 +296,22 @@ def mutate(root, info, story_details, story_file):
         except Exception as e:
             error_message = getattr(e, "message", None)
             raise Exception(error_message if error_message else str(e))
+
+
+class ProcessStory(graphene.Mutation):
+    class Arguments:
+        story_file = Upload(required=True)
+
+    story_contents = graphene.Field(lambda: graphene.List(StoryContentsResponseDTO))
```

Comment:
i think its fine if the return type is just `graphene.Field(lambda: graphene.List(graphene.String))`, the 
id, story_id, line_index, is_deleted isnt necessary

Reply:
Wait don't we need the line index when we display the story preview?